urbit/sur/volt.hoon

```diff
@@ -299,13 +299,37 @@
       [%forward-payment =payreq htlc=update-add-htlc:msg:bolt dest=(unit ship)]
   ==
 ::
++$  chan-info
+  $:  =id:bolt
+      who=ship
+      our=msats
+      his=msats
+      funding-address=(unit address:bc)
+      status=chan-state:bolt
+  ==
++$  pay-info
+  $:  =payreq
+      chan=id:bolt
+      amt=sats:bc
+      pat-p=(unit ship)
+      node-id=(unit @)
+      done=?
+  ==
+::
 +$  update
   $%  [%need-funding-signature temporary-channel-id=@ =address:bc]
       [%need-funding =address:bc =msats]
       [%channel-state =chan-id =chan-state:bolt]
       [%received-payment from=ship =amt=msats]
-      [%new-invoice =payreq]
+      [%new-invoice =payment-request]
       [%invoice-paid =payreq]
       [%payment-result =payreq success=?]
+      [%new-channel =chan-info]
+      [%channel-deleted id=@]
```

Comment:
even if you keep the pattern of sending a sub update for removal of a larval channel, probably want a more specific/semantic name for removing a larval channel vs later on removing a closed channel etc - 'delete' is a little confusing/unprincipled since channels aren't per se deleted